sdk/cosmosdb/cosmos/src/ClientContext.ts

```diff
@@ -29,9 +29,7 @@ import { SessionContainer } from "./session/sessionContainer";
 import { SessionContext } from "./session/SessionContext";
 import { BulkOptions } from "./utils/batch";
 import { sanitizeEndpoint } from "./utils/checkURL";
-import { AzureLogger, createClientLogger } from "@azure/logger";
-
-const logger: AzureLogger = createClientLogger("ClientContext");
+import { defaultLogger } from "./common/logger";
```

Comment:
Are changes in this file related to AAD?

If not can we please scope the changes to just AAD.